openapi.yaml

```diff
@@ -299,6 +299,13 @@ paths:
           schema:
             type: string
           example: Passed
+        - name: status
+          in: query
+          description: |
+            Execution Status for the run. Commonly queried values: 'building','generating','running'
```

Comment:
Wondering if we can describe it better... maybe 
"Test run lifecycle status" ? is that better than "Execution Status for the run."